_posts/2023-04-05-geoserver-2-23-0-released.md

```diff
@@ -0,0 +1,233 @@
+---
+author: Jody Garnett
+date: 2023-04-05
+layout: post
+title: GeoServer 2.23.0 Release
+categories:
+- Announcements
+tags:
+- Release
+release: release_223
+version: 2.23.0
+jira_version: 16885
+---
+
+GeoServer [2.23.0](/release/2.23.0/) release is now available with downloads ([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.23.0/geoserver-2.23.0-bin.zip/download), [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.23.0/geoserver-2.23.0-war.zip/download), [windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.23.0/GeoServer-2.23.0-winsetup.exe/download)), along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.23.0/geoserver-2.23.0-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.23.0/extensions/).
+
+This is a stable release of GeoServer suitable for production systems, made in conjunction with GeoTools 29.0 and GeoWebCache 1.23.0.
+
+Thanks to Jody Garnett (GeoCat) for making this release. Additional thanks to those volunteering to test the release candidate, your assistance is seen and appreciated: Peter Rushforth, Mark Prins, Gabriel Roldan, and Juan Luis.
```

Comment:
Is this correct? I know it was a short testing window :)

@aaime you indicated that extensive testing had been done of the spring framework and spring-security update - can we thank anyone from GeoSolutions here also?